src/components/TabsView/index.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react"
+import Tabs from "@mui/material/Tabs"
+import Tab from "@mui/material/Tab"
+import Typography from "@mui/material/Typography"
+import Box from "@mui/material/Box"
+import { TabsWrapper } from "./styles"
+
+interface TabPanelProps {
+  children?: React.ReactNode
+  dir?: string
+  index: number
+  value: number
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`full-width-tabpanel-${index}`}
+      aria-labelledby={`full-width-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  )
+}
+
+export default function TabsView({ pdfList }: any) {
+  const [value, setValue] = React.useState(0)
```

Comment:
You can import useState only for React
```suggestion
import { useState } from "React"
  const [value, setValue] = useState<number>(0)
```